### PR TITLE
Io refactoring

### PR DESCRIPTION
**Pull request by whaley-ethan**

### CODE REVIEW

File: src/main/java/tic/tac/toe/ConsoleInputOutput.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package tic.tac.toe;
+import java.util.Scanner;
+
+public class ConsoleInputOutput implements InputOutput {
+    public int getInput() {
```
Comment (whaley-ethan) [inline src/main/java/tic/tac/toe/ConsoleInputOutput.java:5]: Should be String -- deferred

File: src/main/java/tic/tac/toe/ConsoleInputOutput.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            returnValue = sc.nextInt();
+        }
+        catch (Exception e) {
+
+        }
+        return returnValue;
+    }
+
+    public void sendOutput(String output) {
+        System.out.println(output);
```
Comment (whaley-ethan) [inline src/main/java/tic/tac/toe/ConsoleInputOutput.java:18]: change to print, not println, include new line with string

File: src/main/java/tic/tac/toe/HumanPlayer.java [hunk 3]
```diff
@@ -1,19 +1,16 @@
 package tic.tac.toe;
 
-
-import java.io.InputStream;
-import java.util.Scanner;
-
 public class HumanPlayer implements Player{
     private boolean didMakeMove = false;
     private char symbol;
+    private InputOutput io;
```
Comment (whaley-ethan) [inline src/main/java/tic/tac/toe/HumanPlayer.java]: rename io to inputOutput for consistancy

File: src/test/java/tic/tac/toe/GameTest.java [hunk 4]
```diff
@@ -8,18 +8,24 @@
...
   @Before
   public void setUp() {
     ui = mock(UI.class);
     board = new Board();
-    game = new Game(ui, board, player1, player2);
+    io = new ConsoleInputOutput();
+    game = new Game(ui, io, board, player1, player2);
   }
 
   @Test
```
Comment (whaley-ethan) [inline src/test/java/tic/tac/toe/GameTest.java:31]: Maybe test that io is invoked as well -- deferred

**Review by ericmeyer — APPROVED** [review]
Squash commits, then 👍 from me!

File: src/main/java/tic/tac/toe/ConsoleInputOutput.java (hunk 2 above)
Comment (ericmeyer) [reply src/main/java/tic/tac/toe/ConsoleInputOutput.java:18]: My initial thought was being slightly surprised that a newline was "inserted" when calling `sendOutput`, however after thinking about it, I am curious at to what @elijahsullivan and @jchung722 think.

**Pull request merged**